app/models/repository_host/base.rb

```diff
@@ -85,22 +85,19 @@ def repository_owner_class
     end
 
     def update_from_host(token = nil)
-      r = self.class.fetch_repo(repository.id_or_name)
-      return unless r.present?
+      repo_data = self.class.fetch_repo(repository.id_or_name)
+      return unless repo_data.present?
 
-      repository.uuid = r[:id] unless repository.uuid.to_s == r[:id].to_s
-      if repository.full_name.downcase != r[:full_name].downcase
-        clash = Repository.host(r[:host_type]).where("lower(full_name) = ?", r[:full_name].downcase).first
+      if repository.full_name.downcase != repo_data.full_name.downcase
+        clash = Repository.host(repo_data.host_type).where("lower(full_name) = ?", repo_data.full_name.downcase).first
```

Comment:
🎱 It looks like `data.full_name.downcase` is used a bunch - could that get abstracted out a `normalized_name` or something, and have the non-standard-name as the odd name out?